### PR TITLE
git-blame-someone-else: add page

### PR DESCRIPTION
**Pull request by marchersimon**
I'm surprised that we didn't already have that page.

### CODE REVIEW

File: pages/common/git-blame-someone-else.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# git blame-someone-else
+
+> Blame someone else for your bad code.
+> More information: <https://github.com/jayphelps/git-blame-someone-else>.
+
+- Change the commiter and author of a commit:
+
+`git blame-someone-else "{{author <email>}}" {{commit}}`
```
Comment (navarroaxel) [inline pages/common/git-blame-someone-else.md:8]:
```suggestion
`git blame-someone-else "{{author someone@example.com}}" {{commit}}`
```

**Comment by CleanMachine1** [comment]
Its cool that Linus committed to the repo :wink: there

File: pages/common/git-blame-someone-else.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/git-blame-someone-else.md:8]: From the GIF in the README, this is actually correct, as it species the email for the `git log`

```
commit 8d854c2d78cb98afdb9f5a73240e06393260b327 (HEAD -> master, origin/master, origin/HEAD)
Author: Jay Phelps <hello@jayphelps.com>
Date:   Tue Jun 12 21:06:40 2018 -0400

```

Comment (navarroaxel) [reply pages/common/git-blame-someone-else.md:8]: Ok, so
```suggestion
`git blame-someone-else "{{author <someone@example.com>}}" {{commit}}`
```

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by Waples — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]
I thought I approved already sorry

**Pull request merged**